doc/release-notes/release-6.sgml

```diff
@@ -63,7 +63,9 @@ This section gives an account of those changes in three categories:
 <sect1>Changes to existing directives<label id="modifieddirectives">
 <p>
 <descrip>
-	<p>There have been no directives changed.
+	<tag>server_cert_fingerprint</tag>
+	<p>The <em>-sha</em> argument has been removed.
+	<p>The support for this parameter has been broken for years. It has a single-dash syntax that violates the double-dash pattern used for other long <em>ACL</em> options. After removal, Squid behavior will remain the same because <em>SHA1</em> is the default (and the only supported) fingerprinting algorithm.
```

Comment:
This removed value was supposedly an optional field in the syntax, thus should be called an "option" instead of a parameter (a mandatory fields in the syntax).

Also, one P tag per setting in the notes please.
```suggestion
	<p>Removed the broken <em>-sha</em> option. <em>SHA1</em> remains the default and only supported fingerprinting algorithm. Configuring it is unnecessary.
```